pkg/planner/core/base/plan_base.go

```diff
@@ -51,10 +51,10 @@ type Plan interface {
 	ID() int
 
 	// TP get the plan type.
-	TP() string
+	TP(...bool) string
 
 	// Get the ID in explain statement
-	ExplainID() fmt.Stringer
+	ExplainID(...bool) fmt.Stringer
```

Comment:
It would be best to add a parameter name to indicate the meaning of this parameter.